farmdata2_modules/fd2_tabs/fd2_example/ui.html

```diff
@@ -396,11 +396,18 @@
           // Would also need to update data in the db via API here.
         },
 
-        cloneEvent(rowsToClone){
-            for(let i = 0; i < rowsToClone.length; i++){
-                clonedRow = structuredClone(this.newTableData[rowsToClone[i]])
-                this.newTableData.push(clonedRow)
-
+        cloneEvent(IDsToClone){
+            for(let i = 0; i < IDsToClone.length; i++){
+                for(let j = 0; j < this.newTableData.length; j++){
+                    if(this.newTableData[j].id == IDsToClone[i]){
+                        clonedRow = structuredClone(this.newTableData[j])
+                        // Prevents buttons sharing same row.id and thus clicking one
+                        // ticks another one 
+                        clonedRow.id = this.newTableData.length + 1
+                        this.newTableData.push(clonedRow)
+                        break
```

Comment:
The `break` is not exiting the conditional, but is exiting the inner `j` for loop.  So it is advancing to the next element of `IDsToClone` without searching the rest of the `tableData`.  This makes sense because it has already found the row.

I am unsure as to why without the break that the `j` look wouldn't just continue until the full `tableData` was searched and then go onto the next `IDsToClone` element and repeat.  

I think this is fine as it is.  You could convert to using a `filter` in place of the inner `j` loop if you prefer.